cvprac/cvp_api.py

```diff
@@ -249,6 +249,87 @@ def get_inventory(self, start=0, end=0, query=''):
                              (query, start, end), timeout=self.request_timeout)
         return data['netElementList']
 
+    def add_device_to_inventory(self, device_ip, parent_name, parent_key):
+        ''' Add the container to the specified parent.
```

Comment:
Should this say "Add the device to the specified parent container." ?

Reply:
Fixed this.